site/content/en/docs/api_sdk/sdk/highlevel-api.md

```diff
@@ -10,7 +10,8 @@ description: ''
 This layer provides high-level APIs, allowing easier access to server operations.
 API includes _Repositories_ and _Entities_. Repositories provide management
 operations for Entitites. Entitites represent separate objects on the server
-(e.g. tasks, jobs etc).
+(e.g. tasks, jobs etc). The key difference from the low-level API is that
```

Comment:
```suggestion
(e.g. tasks, jobs etc). A key difference from the low-level API is that
```

I don't think you could call it _the_ key difference when there are other important differences as well (like representing operations as methods rather than standalone functions).

Reply:
Class methods are not a big improvement over free functions, especially when we speak about this first layer, where apis and models are already grouped into classes. It's not that important in which order we put words, dots and dashes in the function call (`r = p_cls_foo(p, q)` vs `r = p.foo(q)` vs even `p_cls_foo(p, q, r)`). The one real benefit of this is the ability to inherit and overload, but its not something that I'd like to talk about in the docs. I've changed the wording.